common/src/main/java/com/sequenceiq/cloudbreak/util/JaxRSUtil.java

```diff
@@ -71,6 +75,17 @@ private static WebApplicationException handleUnexpectedError(Response response)
         String errorMessage = transformErrorMessage(textResponse);
         String errormsg = "Status: " + response.getStatusInfo().getStatusCode() + ' ' + response.getStatusInfo().getReasonPhrase()
                 + " Response: " + errorMessage;
+
+        try {
+            if (JsonUtil.isValid(errorMessage)) {
+                ClusterProxyError clusterProxyError = JsonUtil.jsonToType(errorMessage, ClusterProxyErrorTypeReference.get());
```

Comment:
what happens if the errormessage can not be converted to ClusterProxyErrorTypeReference

Reply:
IllegalArgumentException will be thrown by JSonUtil and will be catched and we won't retry